### ISSUE TITLE
FlutterIsolatePlugin.h:1:9: fatal error: 'Flutter/Flutter.h' file not found

### ISSUE THREAD

**Issue by abaturyna** [rmawatson/flutter_isolate#60]
Xcode's output:
↳
In file included from /Users/nastya/Dropbox/BUSINESS/Soft/flutter 1.2.13/.pub-cache/hosted/pub.dartlang.org/flutter_isolate-1.0.0+14/ios/Classes/FlutterIsolatePlugin.m:1:
/Users/nastya/Dropbox/BUSINESS/Soft/flutter 1.2.13/.pub-cache/hosted/pub.dartlang.org/flutter_isolate-1.0.0+14/ios/Classes/FlutterIsolatePlugin.h:1:9: fatal error: 'Flutter/Flutter.h' file not found
#import <Flutter/Flutter.h>
^~~~~~~~~~~~~~~~~~~
1 error generated.
Command CompileSwift failed with a nonzero exit code
note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description

My flutter doctor is :

nastya@MacBook-Pro-Nastya ios % flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 1.22.4, on macOS 11.0.1 20B50 darwin-x64, locale ru-BY)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
[✓] Android Studio (version 4.0)
[✓] IntelliJ IDEA Ultimate Edition (version 2020.1.1)
[✓] Connected device (1 available)

• No issues found!
nastya@MacBook-Pro-Nastya ios %

**Comment by suztomo** [comment]
This? https://github.com/flutter/flutter/issues/16036

**Comment by nmfisher** [comment]
@abaturyna I suspect a `flutter clean` will solve your issue.

Given this has been open for some months, I will close for now - if this is still a problem please open a new issue.
